### DISCUSSION TITLE
ED-10014 correct pairing but Failed to register group

### DISCUSSION THREAD

**Discussion by Anpro88** [Koenkk/zigbee2mqtt#25227] [Devices]
Hello everyone,

On Saturday I moved my entire network with 67 devices from openhab zigbee binding to Zigbee2Mqtt.
Except for my 10 buttons, everything works better than before.
Zigbee2Mqtt runs on a PI4 with the latest version of openhabian in 64bit.
I use the Sonoff Zigbee 3.0 USB Dongle-E Plus as a dongle and I installed sonoff_zbdonglee_zigbee_ncp_8.0.2.0_115200_sw_flow.gbl as the firmware.
So far all devices work, except for 9 EcoDim Zigbee 8 button wall switch - white ED-10014 and a Sunricher Zigbee wall controller - 8 button VES-ZB-WAL-012.
These are correctly recognized and also report LQI and battery status, but the keystroke is not reported in the action channel, but a similar error log always appears:

zh:ember: Failed to register group '57857' in multicast table with status=INVALID_STATE.

The group always has its own number but the message is the same - otherwise no information, log in the frontend set to debug.

Please help, I'm desperate.

Best regards
Andreas

**Comment by Damian79** [comment]
Similar setup, getting lots of the same errors, but not sure what is actually faulty. I migrated a month ago and devices tend to be 80% reliable. Maybe my pir’s causing these errors they’re the most temperamental devices. Does the group id correspond to the z2m groups:


[2024-12-18 20:37:11] error: 	zh:ember: Failed to register group '8' in multicast table with status=INVALID_STATE.
[2024-12-18 21:00:41] error: 	z2m: Publish 'set' 'state' to 'KM_Sckt1_Sub' failed: 'Error: ZCL command 0x7cb94c787b4a0000/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Delivery failed for '35191'.)'
[2024-12-18 21:01:39] error: 	zh:ember: Failed to register group '24' in multicast table with status=INVALID_STATE.
[2024-12-18 21:02:09] error: 	zh:ember: Failed to register group '24' in multicast table with status=INVALID_STATE.
[2024-12-18 21:16:05] error: 	zh:ember: Failed to register group '27' in multicast table with status=INVALID_STATE.
[2024-12-18 21:16:07] error: 	zh:ember: Failed to register group '27' in multicast table with status=INVALID_STATE.
[2024-12-18 22:01:01] error: 	zh:ember: Failed to register group '11' in multicast table with status=INVALID_STATE.
[2024-12-18 22:01:19] error: 	zh:ember: Failed to register group '11' in multicast table with status=INVALID_STATE.

**Reply by Damian79** [reply]
I compared these errors with HA logbook and the group id’s do correspond with the group of lights that were turned on at that time, with no issues. So i’m not sure why these errors are reported. Will test further but the devices or groups appear to be working regardless

**Comment by voc0der** [comment]
So, I know I'm coming in from left field here. But a long time ago I tried making a `group` in Z2M, and then added too many devices to it, and got a bunch of errors multicast errors. Then I deleted the group, and forgot about it for a while, but apparently the busted ID was in storage, didn't seem to cleanly reset. (Kinda like Z-Wave if you fail the device pairing). I isolated the issue to Z2M because it was having problems applying scenes without HASS in the picture. I was getting that error about `INVALID_STATE`, and I was also getting weird issues where some devices were linked that shouldn't have been (like a ghost config).

So, anyways, knowing this, I abandoned `16` which was my problem group, made a custom `20` and deleted 16-20 and made them dummies called (DO NOT USE) essentially. Added the devices to my 20 and 21 groups as I originally wanted, everything worked outta the box with MQTT/HASS/Z2M/ reset.

I definitely can't promise this will help you, but it saved me trying to figure out some crazy situation that I was debugging. I had a scene that was turning on lights in other rooms that I had remade several times. Then other times I was doing things with that group, I'd get those multicast errors. Truly bizarre. I'll admit I'm not an expert with MQTT and I do retain my nodes, so it's possible I did this to myself, but hey. I did try remaking all 16 scenes, rebindings, everything. That ID is just cursed with a ghost.

**Comment by Anpro88** [comment]
THX you all for your help - I tried a few more things myself and I found a workaround. I was comparing the 9 Ecodim Buttons to the one Vesternet and I found out the Vesternet was not causing any issues only the Ecodims, the difference was the Versternet had directly after pairing all four buttons bind to the coordinator and when doing so for the Ecodims all the Errors were gone and since then the network is running like a charm. I even got a few old devices I never got to work within openhab, which were recognised and work flawlessly.

**Reply by RStadlmair** [reply]
Hi Anpro88 ! I have the exact same issue and i´m desperate myself. 
New join of ED-10014 works
Configuring works
After a while i get those errors. When i add the group '57857' in Z2M-configuration.yaml it works for a while
after a day or so the problem comes back with a new group ID.

Any hints ?

**Comment by Anpro88** [comment]
Hi!

What permanently fixed it for me was binding each button pair directly to the Coordinator — specifically the endpoints **genLevelCtrl** and **genOnOff**.

Since doing that, I haven't seen the issue come back — running stable since December 2024.

So instead of adding the group ID to the Z2M config as a workaround, go into Z2M → the device → "Bind" tab, and create direct bindings to the Coordinator for both of those clusters. The group workaround seems to only mask the problem temporarily, as you've noticed.

**Important during the binding process:** You need to press the button pair every few seconds while the binding is being set up. The device listens for a response only briefly before going into power saving mode — so keep pressing it to keep it awake until the binding is confirmed.

Hope that helps!

<img width="1238" height="845" alt="image" src="https://github.com/user-attachments/assets/d7051e32-c326-4d36-aa0e-ef08339974fe" />

**Comment by RStadlmair** [comment]
Hi Anpro88 !
Thanks for the answer. Based on your screenshot, i assume you are german speaking, so you can write me back in german as well.
In my panic-mode yesterday i also consulted Discord and git a similar answer, but with creating groups and binding to those groups, plus adding the devices to those groups.
As i have multiple scenes i still want to control the scenes in HA and just avoid the grou-error.

So what i will try now is to bind all the ecoDim Switches to the Coordinator but no lights. You have one light bound to it, i hope thats not mandatory to make it work.

Again thanks for answering, really appreciate it !
R.

**Comment by Anpro88** [comment]
Hi Roman!

To answer your question: No, binding a light to the Coordinator is **not** mandatory. In my screenshot you can see a light bound there, but that's just how my specific setup ended up — it's not what fixed the issue.

The key fix is binding the **switches/button pairs** to the Coordinator on the **genLevelCtrl** and **genOnOff** clusters. The lights themselves don't need to be part of that binding at all.

The reason I have one light bound per room is purely for **fallback/resilience**: if my smart home system (openHAB) crashes or becomes unavailable, I still have at least one light per room that responds directly to the physical switch — as long as there's power, it works without any software in between. But that's a personal design choice, not a requirement for fixing the group error.

So your plan sounds exactly right: bind only the ecoDim switches to the Coordinator and leave the lights out of it. That way Z2M won't throw the group errors, and you can still control your scenes and lights fully through HA as before — the two things are independent of each other.

I prefer to keep forum discussions in English so everyone can benefit — but if you have more questions, feel free to hit me up via DM in German!

Good luck, hope it stays stable for you like it did for me!

Andreas

**Comment by RStadlmair** [comment]
Hi Andreas !
Thanks for that additional reply, i think i understand Zigbee now a little bit better. Will give this a try and and i´m sure it will work. 
Roman

**Comment by RStadlmair** [comment]
Hi Andreas ! 
I have setup all my 4 switches with the above config. HA automations trigger, buts the group error still raises. OK, for me just nasty. I will monotor this and see if it maybe required a reboot or so. Maybe the ZigBee network needs to rescan.

So solution required, any hint appreciated.

R.

**Comment by andrei-lazarov** [comment]
I had this error on SONOFF Dongle-E with their official 8.0.2 firmware.
Updated to 9.0.2 (v2025.12.3-pre1) from Nerivec and it seems the errors are gone.
https://github.com/Nerivec/silabs-firmware-builder/releases
https://mt.xyzroe.cc/
